numpy/core/src/multiarray/scalarapi.c

```diff
@@ -799,7 +799,7 @@ PyArray_Scalar(void *data, PyArray_Descr *descr, PyObject *base)
             Py_INCREF(descr);
             vobj->obval = NULL;
             Py_SIZE(vobj) = itemsize;
-            vobj->flags = NPY_ARRAY_BEHAVED | NPY_ARRAY_OWNDATA;
+            vobj->flags = NPY_ARRAY_CARRAY | NPY_ARRAY_F_CONTIGUOUS | NPY_ARRAY_OWNDATA;
```

Comment:
that should't be necessary, alignment should be handled correctly and if well deal with it when it causes a problem
most platforms don't care about alignment anyway, at worst you have software handling and some slowdown which is irrelevant for scalars.